src/main/sensors/gyro_init.c

```diff
@@ -40,16 +40,46 @@
 #include "drivers/accgyro/accgyro_mpu3050.h"
 #include "drivers/accgyro/accgyro_mpu6050.h"
 #include "drivers/accgyro/accgyro_mpu6500.h"
+
+#ifdef USE_ACCGYRO_BMI160
 #include "drivers/accgyro/accgyro_spi_bmi160.h"
+#endif
+
+#ifdef USE_ACCGYRO_BMI270
 #include "drivers/accgyro/accgyro_spi_bmi270.h"
+#endif
+
+#ifdef USE_GYRO_SPI_ICM20649
 #include "drivers/accgyro/accgyro_spi_icm20649.h"
+#endif
+
+#ifdef USE_ACC_SPI_ICM20689
```

Comment:
Is there a point to some weird naming conventions? Why can't they all follow the same pattern, like ACCGYRO_SPI_XXX?

Reply:
See https://betaflight.com/docs/development/manufacturer/manufacturer-design-guidelines#421-defines-for-gyro-and-acc

- would love to have all using `ACCGYRO_XXX` 
- SPI should be mandatory - no need to include like `ACCGYRO_SPI_XXX`

Reply:
actually _SPI_ part is redundant as all imu should be spi? and accguro is just IMU

Reply:
Need to think about backwards compatibility - as it one of my stated goals.